src/Stripe.net/Infrastructure/Middleware/ParserPlugin/ChargeLevel3Plugin.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Newtonsoft.Json;
+
+namespace Stripe.Infrastructure.Middleware
+{
+    internal class ChargeLevel3Plugin : IParserPlugin
```

Comment:
We really should revisit how plugins work. It's all copy/pasta for the sake of adding one list of items. There has to be a better way

Reply:
Agreed. I took a stab at this in #1218.